src/unity/toolkits/tcmps/mps_graph_trainer.h

```diff
@@ -1,63 +1,44 @@
 #ifndef GRAPH_C_API_H_
 #define GRAPH_C_API_H_
 
-#import "mps_cnnmodule.h"
-#import "mps_graph_cnnmodule.h"
 #import "mps_trainer.h"
-#import <exception>
-
-#define EXPORT __attribute__((visibility("default")))
-
-/*! \brief  macro to guard beginning and end section of all functions */
-#define API_BEGIN() try {
-//\brief every function starts with API_BEGIN();
-//     and finishes with API_END() or API_END_HANDLE_ERROR
-#define API_END()                                                              \
-  }                                                                            \
-  catch (...) {                                                                \
```

Comment:
Do we know how TC will behave now if there's an exception in MPS (or tcmps) code?  
Is it failing somewhat gracefully?

Reply:
I don't see anything on the Python side that inspects the (status code) return values from TCMPS.... I filed #801